test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -168,44 +201,71 @@ function RunUpdateHelpTests
 {
     param (
         [string]$tag = "CI",
-        [switch]$useSourcePath
+        [switch]$useSourcePath,
+        [switch]$Scope
```

Comment:
`$UserScope` is more descriptive than just `$Scope` for a switch and is consistent with previous use

Reply:
Fixed.